docs/content/troubleshooting/csi_ug_troubleshooting_detect_errors.md

```diff
@@ -0,0 +1,21 @@
+# Detecting errors
```

Comment:
could you remind me why we have this page [especially for the controller](https://github.com/IBM/ibm-block-csi-driver/pull/387#discussion_r731054901)?
it seems like duplication to me anyway

Reply:
It shouldn't be duplicated. This is the UG. This update is specifically regarding multi.conf
In any case - as I said in the other thread I don't know why it says for controller specifically. we should resolve the issue there. Anything that needs to be updated for troubleshooting in the UG should be dealt with separately for 1.8.0 release.

Reply:
nothing to do with advanced docs.
it's from https://github.com/IBM/ibm-block-csi-driver/tree/release-1.7

Reply:
@oriyarde  i looked more into this issue. this issue was put together with the https://github.com/IBM/ibm-block-csi-driver/blob/1.7.0_rev3_docs/docs/content/troubleshooting/csi_ug_troubleshooting_logs.md file.
i've now removed it from the docs on this version as well.

Reply:
@oriyarde  i looked more into this issue. this issue was put together with the https://github.com/IBM/ibm-block-csi-driver/blob/1.7.0_rev3_docs/docs/content/troubleshooting/csi_ug_troubleshooting_logs.md file.
i've now removed it from the docs on this version as well.
Done